xonsh/main.py

```diff
@@ -113,7 +113,7 @@ def path_argument(s):
                     metavar='args',
                     help='Additional arguments to the script specified '
                          'by script-file',
-                    nargs='*',
+                    nargs=argparse.REMAINDER,
```

Comment:
Sadly, no.  The `--interactive` flag just means to start up the shell *like* you were in an interactive terminal.  This means to load the config.json and xonshrc files.  Then if a file is provided, the file is executed and the shell executes.  If no file is provided, the shell goes to an interactive terminal.  The reason this could theoretically be useful is if a user has some configuration that a particular script relies on.  It isn't meant for portability, but can be essential in some cases.